components/collapsible-panel/collapsible-panel.js

```diff
@@ -0,0 +1,408 @@
+			});
+			this._intersectionObserver.observe(this.shadowRoot.querySelector('.d2l-collapsible-panel-top-sentinel'));
+		}
+	}
+
+	render() {
+		const classes = {
+			'd2l-collapsible-panel': true,
+			'has-summary': this._hasSummary,
+			'scrolled': this._isScrolled,
+		};
+		const expandCollapseLabel = this.expandCollapseLabel || this.panelTitle;
+
+		return html`
+			<button
+				aria-expanded="${this.expanded}"
+				class="d2l-offscreen"
+				type="button"
+				@click="${this._toggleExpand}"
+				@focus="${this._onFocus}"
+				@blur="${this._onBlur}"
+			>${expandCollapseLabel}</button>
+			<div class="${classMap(classes)}" @click="${this._handlePanelClick}">
+				<div class="d2l-collapsible-panel-top-sentinel"></div>
+				${this._renderHeader()}
+				<d2l-expand-collapse-content ?expanded="${this.expanded}">
+					<div class="d2l-collapsible-panel-content">
+						<slot></slot>
+					</div>
+				</d2l-expand-collapse-content>
+				<d2l-expand-collapse-content ?expanded="${!this.expanded}">
+					<div class="d2l-collapsible-panel-summary">
+						<slot name="summary" @slotchange="${this._handleSummarySlotChange}"></slot>
+					</div>
+				</d2l-expand-collapse-content>
+			</div>
+		`;
+	}
+
+	updated(changedProperties) {
+		if (changedProperties.has('expanded')) {
+			if (!this.expanded) {
+				this._isScrolled = false;
+			}
+		}
+	}
+
+	_handleActionsClick(e) {
+		const actions = this.shadowRoot.querySelector('.d2l-collapsible-panel-header-actions');
+		if (e.target !== actions) {
+			e.stopPropagation();
+		}
+	}
+
+	_handleHeaderClick(e) {
+		if (this.expanded) {
+			this._toggleExpand();
+			e.stopPropagation();
+		}
+	}
+
+	_handleHeaderSecondaryClick(e) {
+		const header = this.shadowRoot.querySelector('.d2l-collapsible-panel-header-secondary');
+		if (e.target !== header) {
+			e.stopPropagation();
+		}
+	}
+
+	_handlePanelClick(e) {
+		if (!this.expanded) {
+			this._toggleExpand();
+			e.stopPropagation();
+		}
+	}
+
+	_handleSummarySlotChange() {
+		this._hasSummary = true;
+	}
+
+	_onBlur() {
+		const element = this.shadowRoot.querySelector('.d2l-collapsible-panel');
+		element.classList.remove('focused');
+	}
+
+	_onFocus() {
+		const element = this.shadowRoot.querySelector('.d2l-collapsible-panel');
+		element.classList.add('focused');
+	}
+
+	_renderHeader() {
+		return html`
+			<div class="d2l-collapsible-panel-header" @click="${this._handleHeaderClick}">
+				<div class="d2l-collapsible-panel-header-primary">
+					${this._renderPanelTitle()}
+					<div class="d2l-collapsible-panel-header-actions" @click="${this._handleActionsClick}">
+						<slot name="actions" @slotchange="${this._handleActionsSlotChange}"></slot>
+					</div>
+					<div class="d2l-collapsible-panel-opener">
+						<d2l-icon-custom size="tier1">
+							<svg xmlns="http://www.w3.org/2000/svg" width="10" height="18" fill="none" viewBox="0 0 10 18">
+								<path stroke="#494c4e" stroke-linejoin="round" stroke-width="2" d="m9 9-8 8V1l8 8Z"/>
```

Comment:
I think this can be `stroke="var(--d2l-color-tungsten)"`.

Reply:
Oh... I could've sworn I tried this and it didn't work. But it does. Thanks 😄